README.md

```diff
@@ -8,7 +8,7 @@ Welcome to the most advanced and feature-full sets of [JAXB](https://github.com/
 
 The project is currently containing the following tools :
 * [JAXB Maven Plugin](#jaxb-maven-plugin)
-* [JAXB Basics Plugins](#jaxb2-basics)
```

Comment:
if you want to change the anchor, you'll need to change the title down there from `JAXB basics` to `JAXB Plugins`